sqldelight-gradle-plugin/src/test/fixtures/invalid-column-type/failure.txt

```diff
@@ -0,0 +1 @@
+Test.sq line 3:8 - mismatched input 'Text' expecting {K_INTEGER, K_REAL, K_TEXT, K_BLOB}
```

Comment:
Yea. Ideally we'd get `expecting {"INTEGER", "REAL", "TEXT", "BLOB"}` but I'm not sure what the best way to do that is.